dis_snek/models/snek/application_commands.py

```diff
@@ -4,6 +4,7 @@
 import re
 from enum import IntEnum
 from typing import TYPE_CHECKING, Callable, Coroutine, Dict, List, Union, Optional, Any
+import typing
```

Comment:
Cant we just do `from typing import Annotated ...`

Reply:
I can (and probably should) import `Annotated` directly.  But I have a strong dislike of importing functions like `get_origin` and `get_args` by name.